common/app/model/meta.scala

```diff
@@ -352,7 +352,7 @@ case class MetaData(
   val sectionId = section map (_.value) getOrElse ""
   lazy val neilsenApid: String = Nielsen.apidFromString(sectionId)
 
-  private val fullAdUnitPath = AdUnitMaker.make(id, adUnitSuffix)
+  private lazy val fullAdUnitPath = AdUnitMaker.make(id, adUnitSuffix)
```

Comment:
Running `TagPagesTest` in isolation (rather than the full suite of frontend tests) was unexpectedly slow for a small set of unit tests - it turns out that this is because every instance of `model.Tag` has `model.MetaData`, which has this field here calculated from `AdUnitMaker.make(id, adUnitSuffix)`.

[`AdUnitMaker.make()`](https://github.com/guardian/frontend/blob/77fe48ed548240d0f6f260aa9937f2995bf9eec2/common/app/common/commercial/AdUnitMaker.scala#L8-L12) is quite slow on the first call, as it references `dfpAccountId` & `dfpAdUnitGuRoot`, which are both pulled in from Guardian configuration, which attempts to load [**every single available parameter**](https://github.com/guardian/frontend/blob/926406850d200b12d4bbc0b599dd8d2cb2712bd9/common/app/common/configuration.scala#L78) from AWS SSM parameter store. This usually takes several seconds, and _requires_ valid AWS credentials to run.

By making this field `lazy`, and because `TagPagesTest` never attempts to read it, we can avoid this big delay while running `TagPagesTest` by itself, avoid the need to grab AWS credentials to run this particular test, and the test now completes in ~300ms, about 10x faster!

Reply:
nice!